examples/agent-sandbox-rl/tests/test_r2egym_adapter.py

```
@@ -0,0 +1,325 @@
+  barrier = threading.Barrier(2)
+
+  def worker(rt, expected):
+    barrier.wait()
+    for _ in range(50):
+      out, _code = rt._run_kubernetes("echo")
+      if out != expected:
+        errors.append((expected, out))
+
+  ta = threading.Thread(target=worker, args=(rt_a, "ns-a"))
+  tb = threading.Thread(target=worker, args=(rt_b, "ns-b"))
+  ta.start(); tb.start(); ta.join(); tb.join()
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)